model/project.go

```diff
@@ -1098,3 +1098,48 @@ func (p *Project) BuildProjectTVPairsWithAlias(alias string) ([]TVPair, []TVPair
 
 	return pairs, displayTaskPairs, err
 }
+
+// Helper function to fetch a group of versions and their associated builds.
```

Comment:
nit, but this should read "FetchVersionsAndAssociatedBuilds is a helper function to..."